handbook/ldap/ldap.md

```diff
@@ -0,0 +1,120 @@
+</figure>
+
+## LDAP URL and Bind User
+
+You must provide a URL and bind user and bind password to establish an authenticated connection between your SystemLink server and LDAP server.
+
+### LDAP URL
+
+The LDAP URL follows a standard scheme. The URL determines the server to connect to, the server's port, a base search distinguished name (DN), and the attribute that determines the SystemLink username for login.
+
+!!! note "Basic LDAP URL Scheme"
+
+    **Structure:** `ldap://<server-dns>:<port>/<target-entry-dn>?<username-attribute>`
+
+    **Example:** `ldap://example.com:389/dc=example,dc=com?sAMAccountName`
+
+    In this example the server is `example.com`, the port is 389, the base search DN is `dc=example,dc=com` and the LDAP attribute used for user login is `sAMAccountName`. 
+
+!!! note "Example LDAP URLs specifying different usernames for the user Jane Doe"
+
+    **URL:** `ldap://example.com/dc=example,dc=com?sAMAccountName`
+    **Username:** jdoe
+
+    **URL:** `ldap://example.com/dc=example,dc=com?userPrincipalName`
+    **Username:** jane.doe@example.com
+
+    <figure>
+      <img src="../../img/ldap-login.png" width="500" />
+      <figcaption>Username `jdoe` is used because the `sAMAccountName` attribute is specified in the LDAP URL.</figcaption>
+    </figure>
+
+If a username attribute is not specified **NI Web Server Configuration** will automatically add `uid` as the attribute. Depending on how your LDAP directory is setup this attribute may or may not be available.
+
+### Bind User
+
+The bind user and bind password are used to authenticate with the LDAP server. Provide a `distinguishedName` or a `userPrincipalName` to specify the bind user.
+
+!!! note "Bind user formats for the user Jane Doe"
+    **distinguishedName:** cn=jdoe,dc=example,dc=com
+
+    **userPrincipalName:** jane.doe@example.com
+
+## Mapping LDAP Groups, Users, and Attributes to Workspaces and Roles
+
+To add a user or collection of users to a workspace and assign a role you must complete the role mapping workflow.
+
+1. Log in to the SystemLink web application with a user mapped to the [**Server Administrator**](https://www.ni.com/documentation/en/systemlink/latest/setup/predefined-roles/) role.
+
+2. Go to **Security** > **Workspaces** and click the gear icon in the top right.
+
+3. Create a new workspace or edit an existing workspace.
+
+4. Go to the **Role Mappings** tab.
+
+5. Click **+MAPPING** and select one of the available LDAP mapping types: *LDAP Group*, *LDAP User*, or *LDAP Attribute*.
+
+6. If you have have selected *LDAP Attribute* enter a valid key and value. For *LDAP User* or *LDAP Group* enter a valid value.
+
+### LDAP Group Mapping
+
+The LDAP Group mapping looks for the values in the LDAP `memberOf` attribute. This attribute's value is an array of group DNs. The value provided to SystemLink must exactly match one value available in the `memberOf` attribute array.
```

Comment:
I don't think we look at `memberOf`. Instead, it'll query the group and look for `member` and `uniqueMember` attributes of the group. It'll also traverse sub-groups up to a depth of 10. Sub-groups are identified via an `objectClass` of `group` (Active Directory-specific), `groupOfNames`, or `groupOfUniqueNames`. See http://httpd.apache.org/docs/2.4/mod/mod_authnz_ldap.html#reqgroup

Reply:
OK thanks for the clarification. I've rewritten this section to reflect this.